README.md

```diff
@@ -108,3 +109,22 @@ Add the following to your project's `.pre-commit-config.yaml` to enable this:
   hooks:
   - id: mdformat
 ~~~
+
+## Code formatter plugins
+
+Mdformat features a plugin system to support formatting of Markdown code blocks where the coding language has been labeled.
+For instance, if [`mdformat-black`](<https://github.com/hukkinj1/mdformat-black>) plugin is installed in the environment,
```

Comment:
perhaps a note/example about how you can achieve this with your own plugin; by adding and an `mdformat.codeformatter` entry point https://packaging.python.org/guides/distributing-packages-using-setuptools/#entry-points

Reply:
Thank you for the review! Yeah makes sense to add this. I'm thinking to maybe add a CONTRIBUTING.md and add this there, in order to not mix up docs intended for users versus devs.

Reply:
Yeh no problem. FYI, since there is currently no CONTRIBUTING.md, the contributing guidelines are directed towards https://github.com/executablebooks/.github/blob/master/CONTRIBUTING.md (see for example the bottom of this page). Since they will now be directed to the repo one, you might want to give them a mention in your CONTRIBUTING.md.

If you didn't notice as well, things like the issue templates and code of conduct are inherited by default from this repo for all others in the organisation.